api_tests/items/recent_public_items.coffee

```diff
@@ -0,0 +1,35 @@
+CONFIG = require 'config'
+__ = CONFIG.universalPath
+_ = __.require 'builders', 'utils'
+should = require 'should'
+{ nonAuthReq, undesiredErr } = __.require 'apiTests', 'utils/utils'
+populate = require '../fixtures/populate'
+recentPublicUrl = '/api/items?action=recent-public'
+
+describe 'items:recent-public', ->
+  it 'should fetch 15 recent-public items', (done)->
+    populate
+      usersCount: 1
+      publicItemsPerUser: 16
+    .then -> nonAuthReq 'get', recentPublicUrl
+    .then (res)->
+      res.items.length.should.equal 15
+
```

Comment:
style note: the lines 15-17 could be turned into a one-liner like the line 14

Reply:
readability concern, as `.should` should is the first things to read in this file